source/MaterialXGenShader/Syntax.cpp

```diff
@@ -182,7 +182,6 @@ ScalarTypeSyntax::ScalarTypeSyntax(const string& name, const string& defaultValu
 
 string ScalarTypeSyntax::getValue(const Value& value, bool /*uniform*/) const
 {
-    MaterialX::Value::ScopedFloatFormatting scopedFloatFormatting(MaterialX::Value::FloatFormatFixed, 5);
```

Comment:
Removed setting of float format in Value class, to avoid having this hard coded to fixed point 5 decimals here, which will be applied for all languages.